tests/unit/test_config.py

```diff
@@ -42,7 +42,7 @@ def _assert_config_read(environment, mock_config):
                     os.environ[env_name] = prev_environment[env_name]
 
     def test_check_for_updates__false(self):
-        for value in [False, "False", "other"]:
+        for value in [False, "False", "other", 0]:
```

Comment:
I actually don't think we need to handle values other than strings, because I don't think it's possible to provide them that way. In that vein we can probably remove `False` and `True`.

Reply:
Like I said, praw.Reddit(arg=True) would cause an error as well.